### ISSUE TITLE
Chinese characters support?

### ISSUE THREAD

**Issue by iChunyu** [textext/textext#277]
First, thank you very much for providing such a good extension!

But, is it possible to support Chinese characters with textext ?

I tried package `ctex` package with `xelatex`, but the output is empty. I'm not sure whether I miss some settings or this extension cannot support Chinese characters by present. 

Could anybody provide some suggestions? Thanks again!

Version: inkscape,v1.0;  textext, v1.3.0.

**Comment by jcwinkler** [comment]
Hi @iChunyu 
It would be helpful if you could first process the manual toolchain as described [in the documentation](https://textext.github.io/textext/usage/troubleshooting.html#manual-use-of-the-toolchain). Please report the results of your investigation. Then, please provide the TeX-File you used (attach it to your post). I guess something is wrong with the error handling.

**Comment by iChunyu** [comment]
Hi @jcwinkler 

Thanks for your suggestion! 

By processing the manual toolchain, I found that I made a very stupid mistake: I installed textext v1.3 on Windows10, but an earlier version on Ubuntu (dual boot, studying Linux now). I re-install the latest version on Ubuntu and the problem is solved.

The mistake was found when I failed to build the `test.tex` with `pdflatex` because 'ctex' seems not to support `pdflatex` on **Linux**. ( Succeed with `xelatex`)

I'm sorry for my mistake. I would close this issue, and, wish you a happy life.
